### PR TITLE
Assignment 4

### PR DESCRIPTION
**Pull request by alvaromontorio**

### CODE REVIEW

File: Assignment4/alvaromontorio-180252/task06.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+g.add((ns.JaneSmith, vcard.FN, Literal("Jane Smith")))
+g.add((ns.Jane, vcard.Given, Literal("Jane")))
+g.add((ns.Smith, vcard.Family, Literal("Smith")))
+for s, p, o in g:
+  print(s,p,o)
+
+"""**TASK 6.5: Add UPM as the university where John Smith works**"""
+
+g.add(ns.UPM, RDF.type, ns.University)
+g.add((ns.JohnSmith, vCard.work, ns.UPM))
```
Comment (dgarijo) [inline Assignment4/alvaromontorio-180252/task06.py:61]: vcard does not define the work property. Please create your own

File: Assignment4/alvaromontorio-180252/task07.py [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+
+
+print("Resultado de 7.2 en SPARQL:")
+from rdflib.plugins.sparql import prepareQuery
+
+q = prepareQuery("""
+SELECT ?s WHERE {
+{?s rdf:type ns:Person.}
+UNION
+{?sub rdfs:subClassOf ns:Person.
```
Comment (dgarijo) [inline Assignment4/alvaromontorio-180252/task07.py:63]: use * to get all the subclass hierarchy of Person

File: Assignment4/alvaromontorio-180252/task07.py [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+print("Resultado de 7.3 en SPARQL:")
+q = prepareQuery("""
+  SELECT 
+    ?s ?p ?o
+  WHERE { 
+    { ?s rdf:type ns:Person.
+    ?s ?p ?o.}
+    UNION
+    {   ?x rdfs:subClassOf ns:Person.
```
Comment (dgarijo) [inline Assignment4/alvaromontorio-180252/task07.py:96]: same as above

**Review by dgarijo — CHANGES_REQUESTED** [review]

**Review by alvaromontorio — COMMENTED** [review]
Changes made

**Review by dgarijo — APPROVED** [review]

**Pull request merged**